message-bus-kafka/docker-compose.kafka-utils.yml

```diff
@@ -20,7 +20,7 @@ services:
     image: quay.io/cloudhut/kminion:master
     hostname: kafka-minion
     environment:
-      KAFKA_BROKERS: kafka:9092
+      KAFKA_BROKERS: kafka-01:9092,kafka-02:9092,kafka-03:9092
```

Comment:
above comment applies

Reply:
Done in [2e20304](https://github.com/jembi/platform/pull/219/commits/2e2030445ea1ac3b00df4545286a138255968e41)